### PR TITLE
APEXCORE-461: Fixed hiding of overridden ports. Refactored unit test.

### PR DESCRIPTION
**Pull request by bhupeshchawda**

### CODE REVIEW

**Comment by bhupeshchawda** [comment]
@siyuanh Please review

File: engine/src/main/java/com/datatorrent/stram/webapp/TypeGraph.java [hunk 1]
```diff
@@ -827,6 +827,16 @@ private void getAllPortsWithAncestor(TypeGraphVertex portVertex, TypeGraphVertex
     }
   }
 
+  private static boolean contains(List<CompactFieldNode> list, CompactFieldNode vertex)
```
Comment (siyuanh) [inline engine/src/main/java/com/datatorrent/stram/webapp/TypeGraph.java]: Can we have another name for this. "contains" seems too common and not very clear here

**Comment by siyuanh** [comment]
@bhupeshchawda looks good to me, once you rename the method, I'll merge it

File: engine/src/main/java/com/datatorrent/stram/webapp/TypeGraph.java (hunk 1 above)
Comment (bhupeshchawda) [inline engine/src/main/java/com/datatorrent/stram/webapp/TypeGraph.java]: Renamed to isNodeInList()

**Pull request merged**